6-AdvancedScenarios/1-call-api-obo/AppCreationScripts/sample.json

```diff
@@ -1,94 +1,116 @@
+        "Level": 300,
+        "Client": "Angular SPA",
+        "Service": ".NET Core web API",
+        "RepositoryUrl": "ms-identity-javascript-angular-tutorial",
+        "Endpoint": "AAD v2.0",
+        "Platform": "javascript",
+        "Languages": [
+            "typescript",
+            "csharp"
+        ],
+        "Products": [
+            "azure-active-directory",
+            "msal-js",
+            "msal-angular",
+            "microsoft-identity-web"
+        ],
+        "Description": "An Angular single-page application authorizing a .NET Core web API to call Microsoft Graph using on-behalf-of flow"
     },
-    {
-      "Id": "client",
-      "Name": "ProfileSPA",
-      "Kind": "SinglePageApplication",
-      "Audience": "AzureADMyOrg",
-      "HomePage": "http://localhost:4200",
-      "ReplyUrls": "http://localhost:4200",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "Microsoft Graph",
-          "DelegatedPermissions": [ "User.Read" ]
-        },
-        {
-					"Resource": "service",
-					"DelegatedPermissions": [ "access_as_user" ]
-				}
-      ],
-      "ManualSteps": [
-        {
-          "Comment": "Navigate to the Manifest page and set the value 'replyUrlsWithType' as 'Spa'."
-        }
-      ]
-    }
-  ],
-  "CodeConfiguration": [
-    {
-      "App": "service",
-      "SettingKind": "JSON",
-      "SettingFile": "\\..\\API\\appsettings.json",
-      "Mappings": [
-        {
-          "key": "Domain",
-          "value": "$tenantName"
-        },
-        {
-          "key": "ClientId",
-          "value": "service.AppId"
-        },
+    "ReadmeSetup": {
+        "CertificateOption": "1"
+    },
+    "AADApps": [
         {
-          "key": "ClientSecret",
-          "value": "service.AppKey"
+            "Id": "service",
+            "Name": "ProfileAPI",
+            "Kind": "WebApi",
+            "Audience": "AzureADMyOrg",
+            "SampleSubPath": "6-AdvancedScenarios\\1-call-api-obo\\API",
+            "PasswordCredentials": "Auto",
+            "Scopes": [
+                "access_as_user"
+            ],
+            "RequiredResourcesAccess": [
+                {
+                    "Resource": "Microsoft Graph",
+                    "DelegatedPermissions": [
+                        "User.Read",
+                        "offline_access"
+                    ]
+                }
+            ],
+            "OptionalClaims": {
+                "AccessTokenClaims": [
+                    "idtyp",
+                    "acct"
+                ]
+            }
         },
         {
-          "key": "TenantId",
-          "value": "$tenantId"
+            "Id": "client",
+            "Name": "ProfileSPA",
+            "Kind": "SinglePageApplication",
+            "Audience": "AzureADMyOrg",
+            "HomePage": "https://localhost:4200",
+            "ReplyUrls": "http://localhost:4200, https://localhost:4200/auth",
```

Comment:
`https://localhost:4200/auth` doesn't match with `http://localhost:4200/auth` in the code.

Reply:
done!